CONTRIBUTING.rst

```diff
@@ -0,0 +1,111 @@
+.. highlight:: shell
+
+============
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
+You can contribute in many ways:
+
+Types of Contributions
+----------------------
+
+Report Bugs
+~~~~~~~~~~~
+
+Report bugs at https://github.com/Pylons/venusian/issues.
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+Fix Bugs
+~~~~~~~~
+
+Look through the GitHub issues for bugs. Anything tagged with "bug"
+is open to whoever wants to implement it.
+
+Implement Features
+~~~~~~~~~~~~~~~~~~
+
+Look through the GitHub issues for features. Anything tagged with "feature"
+is open to whoever wants to implement it.
+
+Write Documentation
+~~~~~~~~~~~~~~~~~~~
+
+venusian could always use more documentation, whether as part of the
+official venusian docs, in docstrings, or even on the web in blog posts,
+articles, and such.
+
+Submit Feedback
+~~~~~~~~~~~~~~~
+
+The best way to send feedback is to file an issue at
+https://github.com/Pylons/venusian/issues.
+
+If you are proposing a feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+Get Started!
+------------
+
+Ready to contribute? Here's how to set up `venusian` for local development.
+
+1. Fork the `venusian` repo on GitHub.
+2. Clone your fork locally::
+
+    $ git clone git@github.com:your_name_here/venusian.git
+
+3. Install your local copy into a virtualenv::
+
+    $ python3 -m venv env
+    $ env/bin/pip install -e .[docs,testing]
```

Comment:
I wouldn't install a local copy. I don't during dev. `tox` will install and run tests in their own virtualenvs across the supported versions of Python.

Reply:
Lots of development requires being able to do things like set breakpoints in code to examine state:  this environment is intended to support that, not automated testing.

Reply:
Even within a `tox` run you can set breakpoints and the like, and now you are actually testing what the end result will be (a wheel) vs potential other things that are in the python path because of the directory you are in. I know we switched to src directory on purpose, but tox makes it cleaner and easier to not accidentally be testing/validating something that is not the same as what will be tested in CI/CD.